### PR TITLE
Refactor bin directory

### PR DESCRIPTION
**Pull request by EugeneHlushko**
**What kind of change does this PR introduce?**

Refactor bin directory to be aligned with code style according to node version support.

**Did you add tests for your changes?**
it is not necessary

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

no
**Other information**

### CODE REVIEW

File: bin/config-yargs.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 var CONFIG_GROUP = "Config options:";
```
Comment (ematipico) [inline bin/config-yargs.js]: I guess also these can be `const`s?

Comment (EugeneHlushko) [reply bin/config-yargs.js]: wow how did they get through 👍

**Comment by EugeneHlushko** [comment]
The failure doesnt look valid :/

**Comment by ematipico** [comment]
Kicked a new build. Probably it's a random failure

**Comment by EugeneHlushko** [comment]
rebased onto master just in case

**Pull request merged**